src/teleop/gui/src/components/GimbalControls.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <div class="wrap">
+        <div v-show='false' id="key">
+          <input v-on:keydown="keymonitordown">
+          <input v-on:keyup="keymonitorup">
+        </div>
+        <p>Gimbal Controls</p>
```

Comment:
Is it necessary to actually put anything on the screen for gimbal controls? If it is, we should explicitly say that the operator should use w-a-s-d to control the gimbal

Reply:
At the moment, the controls gui is only acting as a testing ground for us to make sure our controls work because we don't really have a reason to integrate them into the other guis yet, I only printed that so I knew it was properly integrated in. Once we add this to an actual gui, I wouldn't show anything